RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -452,6 +457,24 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 
    init(iEvent, iSetup);
 
+   std::auto_ptr<reco::MuonTimeExtraMap> muonTimeMap(new reco::MuonTimeExtraMap());
+   reco::MuonTimeExtraMap::Filler filler(*muonTimeMap);
+   std::auto_ptr<reco::MuonTimeExtraMap> muonTimeMapDT(new reco::MuonTimeExtraMap());
+   reco::MuonTimeExtraMap::Filler fillerDT(*muonTimeMapDT);
+   std::auto_ptr<reco::MuonTimeExtraMap> muonTimeMapCSC(new reco::MuonTimeExtraMap());
+   reco::MuonTimeExtraMap::Filler fillerCSC(*muonTimeMapCSC);
+
+   std::auto_ptr<reco::IsoDepositMap> trackDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler trackDepFiller(*trackDepMap);
+   std::auto_ptr<reco::IsoDepositMap> ecalDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler ecalDepFiller(*ecalDepMap);
+   std::auto_ptr<reco::IsoDepositMap> hcalDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler hcalDepFiller(*hcalDepMap);
+   std::auto_ptr<reco::IsoDepositMap> hoDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler hoDepFiller(*hoDepMap);
+   std::auto_ptr<reco::IsoDepositMap> jetDepMap(new reco::IsoDepositMap());
+   reco::IsoDepositMap::Filler jetDepFiller(*jetDepMap);
```

Comment:
`auto_ptr` is now deprecated. You should use `unique_ptr` or `shared_ptr` instead.